test-cases/artifacts/oel81.yaml

```diff
@@ -0,0 +1,20 @@
+ami_db_scylla_user: 'ec2-user'
```

Comment:
The username of oel7.6 image is `clckwrk`. It's changed from oel8?

Reply:
yes it did